### PR TITLE
adds the spec required state command

### PR DESCRIPTION
**Pull request by mikebrow**
Closes issue #589
Signed-off-by: Mike Brown <brownwm@us.ibm.com>

### CODE REVIEW

**Comment by crosbymichael** [comment]
LGTM

File: state.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// +build linux
+
+package main
+
+import (
+	"os"
+	"time"
+
+	"encoding/json"
```
Comment (mrunalp) [inline state.go]: Can you club this with the other standard library imports and in alphabetical order?

File: state.go [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+type cState struct {
+	// Version is the OCI version for the container
+	Version string `json:"ociVersion"`
+	// ID is the container ID
+	ID string `json:"id"`
+	// InitProcessPid is the init process id in the parent namespace
+	InitProcessPid int `json:"pid"`
+	// Bundle is the path on the filesystem to the bundle
+	Bundle string `json:"bundlePath"`
+	// Path to a directory containing the container's root filesystem.
```
Comment (mrunalp) [inline state.go]: Rootfs is a path to ..

File: state.go (hunk 1 above)
Comment (duglin) [inline state.go]: LOL I noticed that too but held my OCD back!  :-)

Comment (mrunalp) [inline state.go]: heh :)

Comment (mikebrow) [inline state.go]: heh.. done :-)

File: state.go (hunk 2 above)
Comment (mrunalp) [inline state.go]: I don't see the change :/

**Comment by mikebrow** [comment]
All issues addressed...

**Comment by mikebrow** [comment]
Hit refresh?

File: state.go (hunk 2 above)
Comment (mikebrow) [inline state.go]: I initially miss-read your request as to the output text not the // comment.  I've updated both now in a second refresh..

**Comment by mrunalp** [comment]
Thanks! LGTM

**Pull request merged**